test/e2e/azure.go

```diff
@@ -0,0 +1,223 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2018-05-01/dns"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"k8s.io/apimachinery/pkg/types"
+
+	configv1 "github.com/openshift/api/config/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	RESOURCE_GROUP = "azure_resourcegroup"
+	SUBSCIPTION_ID = "azure_subscription_id"
+	TENANT_ID      = "azure_tenant_id"
+	CLIENT_ID      = "azure_client_id"
+	CLIENT_SECRET  = "azure_client_secret"
+	REGION         = "azure_region"
+
+	KUBE_SYSTEM_SECRET_NAME = "azure-credentials"
+)
+
+// config represents common config items for Azure DNS and Azure Private DNS
+type cluserConfig struct {
```

Comment:
Why this structure has json tags? I didn't see where it's marshaled/unmarshared in the code.

Reply:
this is taken from external-dns ref, initial aim is to read from file and and parse.
since we already having value & directly assigned.
will remove json tags.